.github/checksum_checker.sh

```diff
@@ -44,12 +44,24 @@ def calculate_sha256(file_path):
             sha256_hash.update(byte_block)
     return sha256_hash.hexdigest()
 
+def manual_safety_check_hf(repo_id):
+    scanResponse = requests.get('https://huggingface.co/api/models/' + repo_id + "/scan")
+    scan = scanResponse.json()
+    if scan['hasUnsafeFile']:
+        return scan
+    return None
+
 download_type, repo_id_or_url = parse_uri(uri)
 
 new_checksum =  None
 
 # Decide download method based on URI type
 if download_type == 'huggingface':
+    # Check if the repo is flagged as dangerous by HF
```

Comment:
can we maybe make sure we delete the model instead?

Models tends to be flagged after upload, and meanwhile we might have added these already.

Reply:
That's a good point. I think that's one of the arguments in favor of the by-default install block... but when I get a chance I'll modify this script to remove

Reply:
I'm not _totally_ sure how to trigger this script off a branch without temporarily publishing a "bad" gallery entry, so please double check my yq command.